### DISCUSSION TITLE
Multi-User Stuttering (VTK)

### DISCUSSION THREAD

**Discussion by joserand** [Kitware/trame#380] [General]
Hi, I'm here again. The problem I encountered this time is that after the docker vtk example ran successfully, I tried to deploy my service, and the operation shown was stuck (the server was running  more lag than I was running directly on a windows computer), what could be the reason for this?
On the left is my Windows computer running, and on the right is a docker multi-user deployment running on the server.

https://github.com/Kitware/trame/assets/89247520/039d651f-bb29-438f-a5ea-f8b2f6ce7079

Thanks for anyone's reply

**Comment by jourdain** [comment]
Are you using remote rendering with OSMesa in docker? Is so, that would explain why.
You need to rely on EGL and make sure you run docker with GPU.

**Reply by joserand** [reply]
I'm sorry to reply you now, I've been preparing for the exam lately and rarely have time to get on GitHub.
Yes, I'm using osema and I'd love to try the egl version, but I can't find vtk-egl in the corresponding index-url location. The vtk-egl build looks a bit difficult, can you give a little bit of reference code on how to build egl-vtk.

**Reply by jourdain** [reply]
- https://discourse.vtk.org/t/pip-packages-with-egl/9745/6
- https://github.com/pyvista/pyvista-wheels
